ci-operator/config/openshift/loki/openshift-loki-main.yaml

```diff
@@ -76,10 +76,10 @@ tests:
     owner: openshift-ci
     product: ocp
     timeout: 1h0m0s
-    version: "4.11"
+    version: "4.12"
   steps:
     dependencies:
-      OO_INDEX: ci-index-loki-operator-bundle
+      OO_BUNDLE: loki-operator-bundle
```

Comment:
This line is different between `main` and the two release branches. Is it maybe the cause of the failure or are the other two files missing this change?